src/app/services/authorization.service.ts

```diff
@@ -46,7 +47,7 @@ export class AuthorizationService {
           console.log('Token: '+ e.newValue);
           this.authToken.value = e.newValue;
           window.removeEventListener('storage', storageChanged, false);
-
+          this.spotifyService.token = e.newValue;
```

Comment:
We are saving this token value in multiple variables/classes, are we using all of them? maybe we need to unifiy them in only one place. (See line 48... this.authToken.value = e.newValue;)